static/src/js/components/MetadataForm/MetadataForm.jsx

```diff
@@ -193,7 +194,7 @@ const MetadataForm = () => {
         conceptType: derivedConceptType,
         metadata: removeEmpty(ummMetadata),
         nativeId,
-        providerId,
+        providerId: fetchedMetadataProviderId || providerId,
```

Comment:
It looks like it other spots we are checking for `conceptId` to be `new` in order to determine if this is a new draft here. I wonder if we should be more explicit and do the same here with a ternary.